gamedata/lua/lua/ui/help/tooltips.lua

```diff
@@ -2028,99 +2028,99 @@ Tooltips = {
-        description = "No Air Experimentals can be built",
+        title = "No Air Experimentals",
+        description = "Players will not be able to build any air experimentals",
     },
-    restricted_units_tactical = {
-        title = "No Tactical missile structures.",
-        description = "Players will not be able to build T2 and T3 tactical missile launching structures.",
+    restricted_units_tml = {
+        title = "No TMLs",
+        description = "Players will not be able to build Tactical Missile Launchers",
     },
     restricted_units_nukes = {
         title = "No Nukes",
-        description = "Players will not be able to build any nuke launching units - launchers, subs or battleships ",
+        description = "Players will not be able to build nuke defences or launchers",
     },
-    restricted_units_T3_Tactical_artillery = {
-        title = "No T3 Artillery structures.",
-        description = "Players will not be able to build T3 Tactical (Barrage) artillery.",
+    restricted_units_T3_tactical_artillery = {
+        title = "No T3 Barrage Artillery",
+        description = "Players will not be able to build T3 Barrage Artillery",
     },
-    restricted_units_T3_Strategic_artillery = {
-        title = "No T3 Strategic Artillery.",
-        description = "Players will not be able to build T3 Strategic artillery.",
+    restricted_units_T3_strategic_artillery = {
+        title = "No T3 Strategic Artillery",
+        description = "Players will not be able to build T3 Strategic Artillery",
     },
     restricted_units_exp_artillery = {
-        title = "No Experimental Artillery",
-        description = "Players will not be able to build T4 Experimental (Mavor, Scathis, Yolona Oss, Salvation) artillery.",
+        title = "No Experimental/T4 Artillery",
+        description = "Players will not be able to build Experimental/T4 Artillery",
     },
-    restricted_units_bubbles = {
-        title = "No Bubbles",
-        description = "Players will not be able to build mobile shield generators or shield defenses",
+    restricted_units_shields = {
+        title = "No Shields",
+        description = "Players will not be able to build mobile or stationary shield generators",
+    },
+    restricted_units_sacu = {
+        title = "No Support Commanders (SACUs)",
+        description = "Players will not be able to build Support Commanders -- NOTE: THIS WILL PREVENT THE BUILDING OF ALMOST ALL EXPERIMENTAL UNITS",
     },
     restricted_units_intel = {
         title = "No Intel Structures",
-        description = "Players will not be able to build radar, sonar and omni installations",
-    },
-    restricted_units_supcom = {
-        title = "No Support Commanders",
-        description = "Players will not be able to build support commanders -- NOTE: THIS WILL PREVENT THE BUILDING OF ALMOST ALL EXPERIMENTAL UNITS",
+        description = "Players will not be able to build Radar, Sonar or Omni installations",

```

Comment:
Regarding the comment about "Optical", I didn't include any reference to Eye of Rhianne, Soothsayer or any of those structures, so I think this description that will be seen in the unit restrictions should be something like:

"Players will not be able to build Radar, Omni, Sonar or special intel structures and units"

Reply:
Agreed. Go forward with this.